assets/src/components/Label.js

```diff
@@ -1,8 +1,28 @@
 import React from 'react'
 
+const alignThreshold = 25
+
+const getLeft = (value) => {
+  const left = value > alignThreshold ? '0' : value + '%'
```

Comment:
Can this simply be for both `getLeft` and `getRight` function?
```
return `value > alignThreshold ? '0' : value + '%'`
```

Reply:
Yes they can.